docs/src/introduction/deploy-and-call-with-cli.md

```diff
@@ -0,0 +1,57 @@
+# Deploy and Call a Sway Contract With the CLI
+
+## Spin Up a Fuel node
+
+In order to deploy the contract, let's spin up a local Fuel node:
+
+```console
+Fuel-core --db-type in-memory
+```
+
+## Deploy `wallet_contract` To Your Local Fuel Node
+
+To deploy `wallet_contract` on your local Fuel node, run
+
+```console
+forc deploy
+```
+
+from the root of the `wallet_contract` directory.
+
+This should produce some output in `stdout` that looks like this:
+
+```console
+  Compiled library "lib-std" with 7 warnings.
```

Comment:
In the most recent version of the compiler with the latest stdlib, there are no more warnings, so now's a good time to remove this line!